library/hpe_icsp_os_deployment.py

```diff
@@ -203,6 +191,7 @@ def main():
             username=dict(required=True, type='str'),
             password=dict(required=True, type='str', no_log=True),
             server_id=dict(required=True, type='str'),
```

Comment:
As you optionally can use the ipAddress as identifier, I guess the server_id could be made required=False